### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Tompage1994**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
```
Comment (djdanielsson) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]: there is a link to feature requests to be on the forums currently that I think needs to get changed.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 ---
 name: Bug report
-about: >
-  Create a bug report. Please test against the latest release before
-  submitting it. For anything else, please use discussions
-  link below.
+about: Create a bug report. Please test against the latest release before submitting
```
Comment (djdanielsson) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: this should not say discussions but should talk about going to the forums for things that are not bug reports or feature requests.

**Review by djdanielsson — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (Tompage1994) [reply .github/ISSUE_TEMPLATE/feature_request.md:2]: I think this changes it

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**